src/app/commands/cmd_save_palette.cpp

```diff
@@ -20,57 +19,62 @@
 #include "app/i18n/strings.h"
 #include "app/modules/palettes.h"
 #include "base/fs.h"
-#include "doc/palette.h"
+#include "new_params.h"
```

Comment:
Use the full path from `src/` dir:
```suggestion
#include "app/commands/new_params.h"
```